weed/server/master_server.go

```diff
@@ -366,14 +366,15 @@ func (ms *MasterServer) OnPeerUpdate(update *master_pb.ClusterNodeUpdate, startF
 					hashicorpRaft.ServerAddress(peerAddress.ToGrpcAddress()), 0, 0)
 			}
 		}
-		if ms.onPeerUpdatDoneCnExist {
-			ms.onPeerUpdatDoneCn <- peerName
+		if ms.onPeerUpdateDoneCnExist {
+			ms.onPeerUpdateDoneCn <- peerName
 		}
 	} else if isLeader {
 		go func(peerName string) {
```

Comment:
Not very familiar with this part. But in general, do not start too many goroutines.